documentation/configuringDynawo/DynawoInputFiles.tex

```diff
@@ -542,6 +542,106 @@ \subsubsection{Specify the local initialization solver parameters}
 </set>
 \end{lstlisting}
 
+\subsubsection{Specify the interactive mode settings}
+
+To run \Dynawo in interactive mode, settings needs to be added to the jobs file.
```

Comment:
Maybe it would be good to add a description of what is the interactive mode somewhere?